### DISCUSSION TITLE
Unable to Install PySCF package

### DISCUSSION THREAD

**Discussion by kishore4362** [pyscf/pyscf#2619] [General]
Building wheel for pyscf (pyproject.toml) ... error
  × Building wheel for pyscf (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [19 lines of output]
.
.
.

 note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyscf
Failed to build pyscf
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (pyscf)
![Screenshot (1)](https://github.com/user-attachments/assets/4457feb3-7e07-4725-afbb-bef60de83981)

I tried so many times to install the package but unable to do that, can anyone please help me with this issue.

**Comment by matthew-hennefarth** [comment]
It seems you are trying to build this on Windows. [PySCF is not supported natively on Windows](https://pyscf.org/user/install.html#how-to-install-pyscf). You can install it through the Windows Subsystem for Linux.

**Comment by verena-neufeld** [comment]
Thank you for your question @kishore4362 , has it been resolved and can this issue be closed?
